packages/components/src/radio-control/index.tsx

```diff
@@ -98,6 +105,16 @@ export function RadioControl(
 						>
 							{ option.label }
 						</label>
+						{ !! option.helpText ? (
+							<Text
+								variant="muted"
+								size={ 12 }
+								id={ `${ id }-${ index }-help` }
+								className="components-radio-control__help-text"
+							>
```

Comment:
Ideally some of these typography/color styles should be shared with the [help styles in BaseControl](https://github.com/WordPress/gutenberg/blob/bced5e0bebb322d6ddb47dafae52fedef9943315/packages/components/src/base-control/styles/base-control-styles.ts#L62) so they stay in sync.

Reply:
Done — although it's annoying that `StyledHelp` applies a top margin. As a low-priority follow-up, we should move the margin somewhere else in `BaseControl` (ideally a flex wrapper ?)